contracts/lbp/LBPManagerFactoryNoAccessControl.sol

```diff
@@ -0,0 +1,158 @@
+/*
```

Comment:
Can you add a commit from the old `contracts-v2` repo, from when you copied the LPB contracts over?